cmd/openshift-install/agent_internal_integration_test.go

```diff
@@ -322,7 +322,8 @@ func checkFileFromInitrdImg(isoPath string, fileName string) error {
 
 			// check if the current cpio files match the required ones
 			for _, f := range files {
-				if f == fileName {
+				matched, err := filepath.Match(filepath.Base(fileName), filepath.Base(f))
```

Comment:
It's not clear to me the reason for stripping out the path from the comparison. The location is relevant, so I think it's be better to avoid using filepath.Base.